This file is a staging area for me to save papers I find but have not yet sorted or read thoroughly. 

## Categories:  

[RNA Biology](#rna-biology)

[Structural Studies](#structural-studies)

[Computation](#computation)

[Miscellaneous](#miscellaneous)

## RNA Biology

**TDP-43 directly inhibits mRNA accumulation in neurites through modulation of mRNA stability**  
Moffatt, Arora, Vaeth, Guzman, Bhardwaj, Hoelscher, Gifford, Russ, Dominguez, Taliaferro  
*The EMBO Journal, December 2025.*  
[[10.1038/s44318-025-00653-4](https://doi.org/10.1038/s44318-025-00653-4)]  
\___\_  

**Replication not required: mRNA vaccines take on non-enveloped viruses**  
Erasmus  
*Molecular Therapy Nucleic Acids, December 2025.*  
[[10.1016/j.omtn.2025.102774](https://doi.org/10.1016/j.omtn.2025.102774)]  
\___\_  

**Theoretical basis for stabilizing messenger RNA through secondary structure design**  
Wayment-Steele, Kim, Choe, Nicol, Wellington-Oguri, Watkins, Parra Sperberg, Huang, Participants, Das  
*Nucleic Acids Research, October 2021.*  
[[10.1093/nar/gkab764](https://doi.org/10.1093/nar/gkab764)]  
\___\_  

**Cell-surface RNA forms ternary complex with RNA-binding proteins and heparan sulfate to recruit immune receptors**  
Li, Joshi, Yin, Wijdeven, Koç, Zijlmans, Santos-Barriopedro, Mei, Wu, Shademan, Zawisza, Bos, Chopra, Tanenbaum, Sharp, Vermeulen, Raz, Joo  
*Molecular Cell, December 2025.*  
[[https://doi.org/10.1016/j.molcel.2025.11.020](https://www.sciencedirect.com/science/article/pii/S1097276525009372)]  
\___\_  

**Synergy between RNA editing and alternative splicing modulates the biological properties of the voltage-gated calcium channel CaV1.3**  
Munyao, Rahman, Wang, Beauvil, Yu, Ruggiu  
*Journal of Biological Chemistry, August 2025.*  
[[10.1016/j.jbc.2025.110426](https://doi.org/10.1016/j.jbc.2025.110426)]  
\___\_  


## Structural Studies

**Atomic-resolution structural information from scattering experiments on macromolecules in solution**  
Hummer  
*,  .*  
[[10.1103/PhysRevE.87.052712](https://link.aps.org/doi/10.1103/PhysRevE.87.052712)]  
\___\_  

**Special issue: Multiscale simulations of DNA from electrons to nucleosomes**  
Maddocks, Dans, Cheatham, Harris, Laughton, Orozco, Pollack, Olson  
*Biophysical Reviews, June 2024.*  
[[10.1007/s12551-024-01204-7](https://doi.org/10.1007/s12551-024-01204-7)]  
\___\_  

**Integrating SHAPE Probing with Direct RNA Nanopore Sequencing Reveals Dynamic RNA Structural Landscapes**  
White Bear, De Bisschop, Lécuyer, Waldispühl  
*, January 2025.*  
[[10.64898/2025.12.14.694260](http://biorxiv.org/content/early/2025/12/16/2025.12.14.694260.abstract)]  
\___\_  

**Untangling or further entangling? Revelation of the complicated world of nucleic acid quadruplex folds**  
Rathinavelan, Sundaresan  
*, January 2025.*  
[[10.64898/2025.12.14.694268](http://biorxiv.org/content/early/2025/12/16/2025.12.14.694268.abstract)]  
\___\_  


## Computation



## Other

### Experimental Biology

**Organelles harbour pH gradients**  
Lee, Chakraborty, Kim, Ali, Mitra, Zajac, Brown, Pincus, Krishnan  
*, January 2025.*  
[[10.64898/2025.12.12.694065](http://biorxiv.org/content/early/2025/12/16/2025.12.12.694065.abstract)]  
\___\_  

**Rotational settings quantize nucleosome movement by chromatin regulators**  
La, Trouth, Ramani, Ramachandran  
*, January 2025.*  
[[10.64898/2025.12.12.693854](http://biorxiv.org/content/early/2025/12/15/2025.12.12.693854.abstract)]  
\___\_  


### Bioinformatics

**Environmental phylogenetics supports a steady diversification of crown eukaryotes starting from the mid Proterozoic**  
Sandin, Cohen, Morlon, Burki  
*, January 2025.*  
[[10.64898/2025.12.12.693929](http://biorxiv.org/content/early/2025/12/14/2025.12.12.693929.abstract)]  
\___\_  

**Sequence-Based Generative AI-Guided Design of Versatile Tryptophan Synthases**  
Lambert, Tavakoli, Dharuman, Yang, Bhethanabotla, Kaur, Hill, Ramanathan, Anandkumar, Arnold  
*, January 2025.*  
[[10.1101/2025.08.30.673177](http://biorxiv.org/content/early/2025/08/30/2025.08.30.673177.abstract)]  
\___\_  

**Limitations of the refolding pipeline for &lt;em&gt;de novo&lt;/em&gt; protein design**  
Korbeld, Viliuga, Fürst  
*, January 2025.*  
[[10.64898/2025.12.09.693122](http://biorxiv.org/content/early/2025/12/11/2025.12.09.693122.abstract)]  
\___\_  

**&lt;em&gt;CUT&amp;amp;ID&lt;/em&gt; for simultaneous profiling of protein-DNA and protein-protein interactions**  
Nordin, Cantù  
*, January 2025.*  
[[10.64898/2025.12.11.693662](http://biorxiv.org/content/early/2025/12/12/2025.12.11.693662.abstract)]  
\___\_  


### Miscellaneous

**Illuminating Solution Responses of a LOV Domain Protein with Photocoupled Small-Angle X-Ray Scattering**  
Lamb, Zoltowski, Pabit, Li, Crane, Pollack  
*Journal of Molecular Biology, November 2009.*  
[[https://doi.org/10.1016/j.jmb.2009.08.045](https://www.sciencedirect.com/science/article/pii/S0022283609010535)]  
\___\_  

**Increasing protein stability by inferring substitution effects from high-throughput experiments**  
Norrild, Johansson, O’Shea, Morth, Lindorff-Larsen, Winther  
*Cell Reports Methods, November 2022.*  
[[10.1016/j.crmeth.2022.100333](https://doi.org/10.1016/j.crmeth.2022.100333)]  
\___\_  


**Discovering Glycosylation-Dependent Protein Function by Thermal Proteome Profiling**  
Hevler, Sojitra, Caval, Schoof, Mateus, Bertozzi  
*, January 2025.*  
[[10.64898/2025.12.06.692499](http://biorxiv.org/content/early/2025/12/07/2025.12.06.692499.abstract)]  
\___\_  

**Satellite megaconstellations will threaten space-based astronomy**  
Borlaff, Marcum, Howell  
*Nature, December 2025.*  
[[10.1038/s41586-025-09759-5](https://doi.org/10.1038/s41586-025-09759-5)]  
\___\_  

**evo3D: a generalised framework for structure-informed evolutionary analysis, implemented in R**  
Broyles, He  
*, January 2025.*  
[[10.64898/2025.12.12.694041](http://biorxiv.org/content/early/2025/12/14/2025.12.12.694041.abstract)]  
\___\_  

**Scikit-bio: a fundamental Python library for biological omic data analysis**  
Aton, McDonald, Cañardo Alastuey, Azom, Batra, Bezshapkin, Bolyen, Cagle, Caporaso, Debelius, Gorlick, Hamsanipally, Hunger, Keluskar, Liao, Lu, Navas-Molina, Pitman, Rideout, Sazonov, Sathappan, Schwarzberg Lipson, Sfiligoi, Tapo, Vázquez-Baeza, Wu, Xu, Ye, Zhao, Knight, Morton, Zhu  
*Nature Methods, December 2025.*  
[[10.1038/s41592-025-02981-z](https://doi.org/10.1038/s41592-025-02981-z)]  
\___\_  

**Benchmarking datasets for machine learning in protein function prediction**  
Jang, Qin, Wang, Kornmann  
*, January 2025.*  
[[10.64898/2025.12.17.694800](http://biorxiv.org/content/early/2025/12/19/2025.12.17.694800.abstract)]  
\___\_  

**Structural basis of undecaprenyl phosphate glycosylation leading to polymyxin resistance in Gram-negative bacteria**  
Ashraf, Bunoro-Batista, Ansell, Punetha, Rosario-Garrido, Firlar, Kaelber, Stansfeld, Petrou  
*Nature Communications, December 2025.*  
[[10.1038/s41467-025-65968-6](https://doi.org/10.1038/s41467-025-65968-6)]  
\___\_  

**O-GlcNAcylation and an importin-β radial gradient keep the FG barrier liquid in live-cell nuclear pores**  
Yu, Heidari, Palacio-Rodriguez, Ruan, Mingu, Hidayat, Bode, Sikora, Hummer, Lemke  
*, January 2025.*  
[[10.64898/2025.12.09.693204](http://biorxiv.org/content/early/2025/12/11/2025.12.09.693204.abstract)]  
\___\_  

**Atomic models of the Toxoplasma cell invasion machinery**  
Zeng, Fu, Qian, Huang, Niu, Beatty, Brown, Sibley, Zhang  
*Nature Structural & Molecular Biology, December 2025.*  
[[10.1038/s41594-025-01728-w](https://doi.org/10.1038/s41594-025-01728-w)]  
\___\_  

**Sequence and structural determinants of efficacious &lt;em&gt;de novo&lt;/em&gt; chimeric antigen receptors**  
Chow, Chu, Li, Nalbant, Dozic, Kida, Lareau  
*, January 2025.*  
[[10.64898/2025.12.12.694033](http://biorxiv.org/content/early/2025/12/13/2025.12.12.694033.abstract)]  
\___\_  

**Beyond RNA modification: a novel role for tRNA modifying enzyme in oxidative stress response and metabolism**  
Fruchard, Sudol, Rouard, Treffkorn-Maurau, Hardy, Bos, Duchateau, Giai Gianetto, Matondo, Bonhomme, Thuillier, Marchand, Motorin, Bregeon, Mazel, Hamdane, Baharoglu  
*Nucleic Acids Research, December 2025.*  
[[10.1093/nar/gkaf1276](https://doi.org/10.1093/nar/gkaf1276)]  
\___\_  

**The Gene Ontology knowledgebase in 2026**  
The Gene Ontology Consortium   
*Nucleic Acids Research, December 2025.*  
[[10.1093/nar/gkaf1292](https://doi.org/10.1093/nar/gkaf1292)]  
\___\_  

**AlphaFold predictions are valuable hypotheses and accelerate but do not replace experimental structure determination**  
Terwilliger, Liebschner, Croll, Williams, McCoy, Poon, Afonine, Oeffner, Richardson, Read, Adams  
*Nature Methods, January 2024.*  
[[10.1038/s41592-023-02087-4](https://doi.org/10.1038/s41592-023-02087-4)]  
\___\_  

**LLM use in scholarly writing poses a provenance problem**  
Earp, Yuan, Koplin, Porsdam Mann  
*Nature Machine Intelligence, December 2025.*  
[[10.1038/s42256-025-01159-8](https://doi.org/10.1038/s42256-025-01159-8)]  
\___\_  

**Cold and lithium delay forgetting of olfactory memories in Caenorhabditis elegans**  
Landschaft-Berliner, Goldstein, Teichman, Anava, Gingold, Salzberg, Rieger, Levy, Pechuk, Setty, Agarwal, Sagi, Cohen, Nikelshparg, Ben-Zvi, Miranda-Vizuete, Zaidel-Bar, Oren-Suissa, Rechavi  
*Nature Neuroscience, December 2025.*  
[[10.1038/s41593-025-02143-6](https://doi.org/10.1038/s41593-025-02143-6)]  
\___\_  

**IDR searcher: a search engine solution for public image resource**  
Mohamed, Moore, Lindner, Moore, Swedlow, Wong, Burel  
*, January 2025.*  
[[10.64898/2025.12.16.694566](http://biorxiv.org/content/early/2025/12/18/2025.12.16.694566.abstract)]  
\___\_  

**Chromatin buffers torsional stress during transcription**  
Qian, Lubkowska, Zhang, Tan, Hong, Jia, Fulbright, Inman, Kay, Jeong, Hauk, Gotte, Berger, Kashlev, Wang  
*Science,  .*  
[[10.1126/science.adv0134](https://doi.org/10.1126/science.adv0134)]  
\___\_  

**Single-particle tracking reveals heterogeneous PIEZO1 diffusion**  
Ly, Freites, Bertaccini, Evans, Dickinson, Tobias, Pathak  
*Biophysical Journal, December 2025.*  
[[10.1016/j.bpj.2025.01.010](https://doi.org/10.1016/j.bpj.2025.01.010)]  
\___\_  

**BioChemAIgent: An AI-driven Protein Modeling and Docking Framework for Structure-Based Drug Discovery**  
Yousefi, Laubach, Heins, Testa, Gersting, Bonn  
*, January 2025.*  
[[10.64898/2025.12.17.694892](http://biorxiv.org/content/early/2025/12/19/2025.12.17.694892.abstract)]  
\___\_  

**AlphaFold3 and Intrinsically Disordered Proteins: Reliable Monomer Prediction, Unpredictable Multimer Performance**  
Dao, Ghent, Uversky, Rahman  
*, January 2025.*  
[[10.64898/2025.12.05.691730](http://biorxiv.org/content/early/2025/12/10/2025.12.05.691730.abstract)]  
\___\_  

**Mechanism of cotranslational modification of histones H2A and H4 by MetAP1 and NatD**  
Yudin, Jaskolowski, Fan, Burg, Chandrasekar, Lentzsch, Scaiola, Bothe, Deuerling, Gamerdinger, Shan, Ban  
*Science Advances,  .*  
[[10.1126/sciadv.aeb1017](https://doi.org/10.1126/sciadv.aeb1017)]  
\___\_  

**Democratizing protein language model training, sharing and collaboration**  
Su, Li, Tao, Han, He, Dai, Yuan, Gao, Si, Zhang, Zhou, Shan, Zhou, Chang, Jiang, Ma, Gitter, Mirdita, Yang, Notin, Marks, Chatterjee, Singh, Romero, Heinzinger, Liu, Zheng, Li, Zeng, Shen, Chai, Ju, Ferruz, Glasgow, Kim, Snow, Ntranos, Yang, Hong, Gao, Si, Bronstein, Chang, Yuan, Su, Li, Tao, Han, He, Dai, Zhang, Zhou, Shan, Zhou, Jiang, Gao, Zhang, Fan, Tao, Cheng, Zheng, Chen, Long, Kong, Pu, Guan, Zhang, Li, Steinegger, Ovchinnikov, Yuan, The OPMC  
*Nature Biotechnology, October 2025.*  
[[10.1038/s41587-025-02859-7](https://doi.org/10.1038/s41587-025-02859-7)]  
\___\_  

**PXDesign: Fast, Modular, and Accurate De Novo Design of Protein Binders**  
Team, Ren, Sun, Guan, Liu, Gong, Wang, Wang, Cai, Ma, Zhang, Liu, Zhang, Chen, Xiao  
*, January 2025.*  
[[10.1101/2025.08.15.670450](http://biorxiv.org/content/early/2025/12/17/2025.08.15.670450.abstract)]  
\___\_  

**Accelign: a GPU-based Library for Accelerating Pairwise Sequence Alignment**  
Kallenborn, Dabbaghie, Steinegger, Schmidt  
*, January 2025.*  
[[10.64898/2025.12.17.694868](http://biorxiv.org/content/early/2025/12/19/2025.12.17.694868.abstract)]  
\___\_  


**Large scale prospective evaluation of co-folding across 557 Mac1-ligand complexes and three virtual screens**  
Kim, Correy, Hall, Rachman, Mailhot, Togo, Gonciarz, Jaishankar, Neitz, Hantz, Doruk, Stevens, Diolaiti, Reid, Gopalkrishnan, Krogan, Renslo, Ashworth, Shoichet, Fraser  
*, January 2025.*  
[[10.64898/2025.12.25.696505](http://biorxiv.org/content/early/2025/12/28/2025.12.25.696505.abstract)]  
\___\_  

**Cell-surface RNA forms ternary complex with RNA-binding proteins and heparan sulfate to recruit immune receptors**  
Li, Joshi, Yin, Wijdeven, Koç, Zijlmans, Santos-Barriopedro, Mei, Wu, Shademan, Zawisza, Bos, Chopra, Tanenbaum, Sharp, Vermeulen, Raz, Joo  
*Molecular Cell, December 2025.*  
[[10.1016/j.molcel.2025.11.020](https://doi.org/10.1016/j.molcel.2025.11.020)]  
\___\_  

**Template-based RNA structure prediction advanced through a blind code competition**  
Lee, He, Oda, Rao, Kim, Kim, Kim, Heng, Kowerko, Li, Nguyen, Sampathkumar, Enrique Gomez, Chen, Yoshizawa, Kuraishi, Ogawa, Zou, Paullier, Zhao, Chen, Hsu, Hirano, Gezelle, Haack, Hong, Jadhav, Koirala, Kretsch, Lewicka, Li, Marcia, Piccirilli, Rudolfs, Srivastava, Steckelberg, Su, Toor, Wang, Yang, Zhang, Zou, Baker, Chen, Chiu, Demkin, Favor, Hummer, Joshi, Kryshtafovych, Kucukbenli, Miao, Moult, Munley, Reade, Viel, Westhof, Zhang, Das  
*, January 2025.*  
[[10.64898/2025.12.30.696949](http://biorxiv.org/content/early/2025/12/30/2025.12.30.696949.abstract)]  
\___\_  

**Predicting small molecule–RNA interactions without RNA tertiary structures**  
Fei, Wang, Zhang, Shan, Cai, Ma, Wang, Zhang  
*Nature Biotechnology, January 2026.*  
[[10.1038/s41587-025-02942-z](https://doi.org/10.1038/s41587-025-02942-z)]  
\___\_  

**Generalized design of sequence–ensemble–function relationships for intrinsically disordered proteins**  
Krueger, Brenner, Shrinivas  
*Nature Computational Science, October 2025.*  
[[10.1038/s43588-025-00881-y](https://doi.org/10.1038/s43588-025-00881-y)]  
\___\_  

**ChiraKit: an online tool for the analysis of circular dichroism spectroscopy data**  
Burastero, Jones, Defelipe, Zavrtanik, Hadži, Hoffmann, Garcia-Alai  
*Nucleic Acids Research, July 2025.*  
[[10.1093/nar/gkaf350](https://doi.org/10.1093/nar/gkaf350)]  
\___\_  

**Visualizing a viral genome with contrast variation small angle X-ray scattering**  
San Emeterio, Pollack  
*Journal of Biological Chemistry, November 2020.*  
[[https://doi.org/10.1074/jbc.RA120.013961](https://www.sciencedirect.com/science/article/pii/S0021925817504214)]  
\___\_  

**CAML: Commutative Algebra Machine Learning─A Case Study on Protein–Ligand Binding Affinity Prediction**  
Feng, Suwayyid, Zia, Wee, Hozumi, Chen, Wei  
*J. Chem. Inf. Model., July 2025.*  
[[10.1021/acs.jcim.5c00940](https://doi.org/10.1021/acs.jcim.5c00940)]  
\___\_  
