## Categories:  

[Solution scattering](#solution-scattering)  

[Fluorescence](#fluorescence)

[Simulation](#simulation)



## Solution Scattering

**Atomic-resolution structural information from scattering experiments on macromolecules in solution**  
Hummer  
*,  .*  
[[10.1103/PhysRevE.87.052712](https://link.aps.org/doi/10.1103/PhysRevE.87.052712)]  
\___\_  

**Illuminating Solution Responses of a LOV Domain Protein with Photocoupled Small-Angle X-Ray Scattering**  
Lamb, Zoltowski, Pabit, Li, Crane, Pollack  
*Journal of Molecular Biology, November 2009.*  
[[https://doi.org/10.1016/j.jmb.2009.08.045](https://www.sciencedirect.com/science/article/pii/S0022283609010535)]  
\___\_  

**Visualizing a viral genome with contrast variation small angle X-ray scattering**  
San Emeterio, Pollack  
*Journal of Biological Chemistry, November 2020.*  
[[https://doi.org/10.1074/jbc.RA120.013961](https://www.sciencedirect.com/science/article/pii/S0021925817504214)]  
\___\_  

## Fluorescence

## Spectroscopy

**ChiraKit: an online tool for the analysis of circular dichroism spectroscopy data**  
Burastero, Jones, Defelipe, Zavrtanik, Hadži, Hoffmann, Garcia-Alai  
*Nucleic Acids Research, July 2025.*  
[[10.1093/nar/gkaf350](https://doi.org/10.1093/nar/gkaf350)]  
\___\_  
## Simulation

**Physics-driven coarse-grained model for biomolecular phase separation with near-quantitative accuracy**  
Jerelle A. Joseph, Aleks Reinhardt, Anne Aguirre, Pin Yu Chew, Kieran O. Russell, Jorge R. Espinosa, Adiran Garaizar & Rosana Collepardo-Guevara  
*Nature Computer Science, November 2021.*    
[[10.1038/s43588-021-00155-3](https://doi.org/10.1038/s43588-021-00155-3)]  
\__mpipi_jacsRefMay2025_\_  

## Bioinformatics

**Scikit-bio: a fundamental Python library for biological omic data analysis**  
Aton, McDonald, Cañardo Alastuey, Azom, Batra, Bezshapkin, Bolyen, Cagle, Caporaso, Debelius, Gorlick, Hamsanipally, Hunger, Keluskar, Liao, Lu, Navas-Molina, Pitman, Rideout, Sazonov, Sathappan, Schwarzberg Lipson, Sfiligoi, Tapo, Vázquez-Baeza, Wu, Xu, Ye, Zhao, Knight, Morton, Zhu  
*Nature Methods, December 2025.*  
[[10.1038/s41592-025-02981-z](https://doi.org/10.1038/s41592-025-02981-z)]  
\___\_  

**The Gene Ontology knowledgebase in 2026**  
The Gene Ontology Consortium   
*Nucleic Acids Research, December 2025.*  
[[10.1093/nar/gkaf1292](https://doi.org/10.1093/nar/gkaf1292)]  
\___\_  


**evo3D: a generalised framework for structure-informed evolutionary analysis, implemented in R**  
Broyles, He  
*, January 2025.*  
[[10.64898/2025.12.12.694041](http://biorxiv.org/content/early/2025/12/14/2025.12.12.694041.abstract)]  
\___\_  

**IDR searcher: a search engine solution for public image resource**  
Mohamed, Moore, Lindner, Moore, Swedlow, Wong, Burel  
*, January 2025.*  
[[10.64898/2025.12.16.694566](http://biorxiv.org/content/early/2025/12/18/2025.12.16.694566.abstract)]  
\___\_  

**Accelign: a GPU-based Library for Accelerating Pairwise Sequence Alignment**  
Kallenborn, Dabbaghie, Steinegger, Schmidt  
*, January 2025.*  
[[10.64898/2025.12.17.694868](http://biorxiv.org/content/early/2025/12/19/2025.12.17.694868.abstract)]  
\___\_  

## Machine-learning Methods

**AlphaFold3 and Intrinsically Disordered Proteins: Reliable Monomer Prediction, Unpredictable Multimer Performance**  
Dao, Ghent, Uversky, Rahman  
*, January 2025.*  
[[10.64898/2025.12.05.691730](http://biorxiv.org/content/early/2025/12/10/2025.12.05.691730.abstract)]  
\___\_  

**AlphaFold predictions are valuable hypotheses and accelerate but do not replace experimental structure determination**  
Terwilliger, Liebschner, Croll, Williams, McCoy, Poon, Afonine, Oeffner, Richardson, Read, Adams  
*Nature Methods, January 2024.*  
[[10.1038/s41592-023-02087-4](https://doi.org/10.1038/s41592-023-02087-4)]  
\___\_  

**Limitations of the refolding pipeline for &lt;em&gt;de novo&lt;/em&gt; protein design**  
Korbeld, Viliuga, Fürst  
*, January 2025.*  
[[10.64898/2025.12.09.693122](http://biorxiv.org/content/early/2025/12/11/2025.12.09.693122.abstract)]  
\___\_  

**BioChemAIgent: An AI-driven Protein Modeling and Docking Framework for Structure-Based Drug Discovery**  
Yousefi, Laubach, Heins, Testa, Gersting, Bonn  
*, January 2025.*  
[[10.64898/2025.12.17.694892](http://biorxiv.org/content/early/2025/12/19/2025.12.17.694892.abstract)]  
\___\_  

**CAML: Commutative Algebra Machine Learning─A Case Study on Protein–Ligand Binding Affinity Prediction**  
Feng, Suwayyid, Zia, Wee, Hozumi, Chen, Wei  
*J. Chem. Inf. Model., July 2025.*  
[[10.1021/acs.jcim.5c00940](https://doi.org/10.1021/acs.jcim.5c00940)]  
\___\_  

**Democratizing protein language model training, sharing and collaboration**  
Su, Li, Tao, Han, He, Dai, Yuan, Gao, Si, Zhang, Zhou, Shan, Zhou, Chang, Jiang, Ma, Gitter, Mirdita, Yang, Notin, Marks, Chatterjee, Singh, Romero, Heinzinger, Liu, Zheng, Li, Zeng, Shen, Chai, Ju, Ferruz, Glasgow, Kim, Snow, Ntranos, Yang, Hong, Gao, Si, Bronstein, Chang, Yuan, Su, Li, Tao, Han, He, Dai, Zhang, Zhou, Shan, Zhou, Jiang, Gao, Zhang, Fan, Tao, Cheng, Zheng, Chen, Long, Kong, Pu, Guan, Zhang, Li, Steinegger, Ovchinnikov, Yuan, The OPMC  
*Nature Biotechnology, October 2025.*  
[[10.1038/s41587-025-02859-7](https://doi.org/10.1038/s41587-025-02859-7)]  
\___\_  

## High throughput bio

**&lt;em&gt;CUT&amp;amp;ID&lt;/em&gt; for simultaneous profiling of protein-DNA and protein-protein interactions**  
Nordin, Cantù  
*, January 2025.*  
[[10.64898/2025.12.11.693662](http://biorxiv.org/content/early/2025/12/12/2025.12.11.693662.abstract)]  
\___\_  

**Increasing protein stability by inferring substitution effects from high-throughput experiments**  
Norrild, Johansson, O’Shea, Morth, Lindorff-Larsen, Winther  
*Cell Reports Methods, November 2022.*  
[[10.1016/j.crmeth.2022.100333](https://doi.org/10.1016/j.crmeth.2022.100333)]  
\___\_  

## Writing

**LLM use in scholarly writing poses a provenance problem**  
Earp, Yuan, Koplin, Porsdam Mann  
*Nature Machine Intelligence, December 2025.*  
[[10.1038/s42256-025-01159-8](https://doi.org/10.1038/s42256-025-01159-8)]  
\___\_  

